src/gui/editors/AutomationEditor.cpp

```diff
@@ -615,6 +667,36 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 
 
 
+void AutomationEditor::mouseDoubleClickEvent(QMouseEvent * mouseEvent)
+{
+	if (!validPattern()) { return; }
+
+	// If we double clicked inside the AutomationEditor viewport
+	if (mouseEvent->y() > TOP_MARGIN && mouseEvent->x() >= VALUES_WIDTH)
```

Comment:
Style suggestion:

Same inverse guard as above.
```suggestion
	if (mouseEvent->y() <= TOP_MARGIN || mouseEvent->x() < VALUES_WIDTH) { return; }
```
Then indent the function left once.